config/env.config.js

```diff
@@ -69,26 +71,38 @@ const RedirectPlugin = function(options) {
                             if (match && match.length > 1) {
                                 return match[1];
                             }
-                            return "";
+                            return null;
                         };
-                        var originUrl = decodeURIComponent(getParameterByName("originEnvironment"));
-                        var newUrl = window.localStorage.getItem("environmentConfig");
-                        if (newUrl && !window.location.href.match(newUrl) && originUrl !== newUrl) {
+                        var originUrl = getParameterByName("originEnvironment");
+                        var newUrl = getParameterByName("newEnvironment");
+                        if (newUrl) {
+                            newUrl = decodeURIComponent(newUrl);
+                            if (window.location.href.match(newUrl)) {
+                                window.localStorage.removeItem("environmentConfig");
+                            } else {
+                                window.localStorage.setItem("environmentConfig", newUrl);
```

Comment:
to follow the naming convention of the other local-storage items, can we call it `playground_environment_config`.  And could you make it a constant?  Also, I don't know if `config` is quite the right word (I imagine config to be a set of properties as a JSON object or something).  Maybe just a `playground_environment_redirect_url`, or similar?